<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mercurial::Internal::OptionsPage</class>
 <widget class="QWidget" name="Mercurial::Internal::OptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="mercurialCommandLabel">
         <property name="text">
          <string>Mercurial Command:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="Core::Utils::PathChooser" name="commandChooser" native="true"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="showLogEntriesLabel">
         <property name="text">
          <string>Show Log Entries:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="logEntriesCount">
         <property name="toolTip">
          <string>The number of recent commit logs to show, choose 0 to see all enteries</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="timeoutSecondsLabel">
         <property name="text">
          <string>Timeout (Seconds):</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="timeout">
         <property name="value">
          <number>30</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="promptOnSubmitLabel">
         <property name="text">
          <string>Prompt On Submit</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QCheckBox" name="promptOnSubmitCheckBox">
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="defaultUsernameLabel">
         <property name="toolTip">
          <string>Username to use by default on commit.</string>
         </property>
         <property name="text">
          <string>Default Username:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="defaultUsernameLineEdit">
         <property name="toolTip">
          <string>Username to use by default on commit.</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="defaultEmailLabel">
         <property name="toolTip">
          <string>Email to use by default on commit.</string>
         </property>
         <property name="text">
          <string>Default Email:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="defaultEmailLineEdit">
         <property name="toolTip">
          <string>Email to use by default on commit.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Core::Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
